src/04_printing.py

```diff
@@ -12,6 +12,10 @@
 # y, and z:
 # x is 10, y is 2.25, z is "I like turtles!"
 
+print ('x: %2d, y: %5.2f, z: %s' %(x, y, z))
+
 # Use the 'format' string method to print the same thing
+print ('x: {0:2d}, y: {1:5.2f}, z: {2:s}'.format(x, y, z))
 
-# Finally, print the same thing using an f-string
\ No newline at end of file
+# Finally, print the same thing using an f-string
+print(f"x: {x}, y: {y:5.2f}, z: {z} ")
```

Comment:
Great job with printing....I see you've mastered f"string. be aware it only works with Python 3